modules/end-user-guide/pages/selecting-a-workspace-ide.adoc

```diff
@@ -9,6 +9,8 @@
 
 The default in-browser IDE in a new workspace is Microsoft Visual Studio Code - Open Source.
 
+NOTE: Because the {prod-short} build of Microsoft Visual Studio Code - Open Source supports branding, your organization might be using a branded build.
```

Comment:
```suggestion
NOTE: Because the {prod-short} build of Microsoft Visual Studio Code - Open Source supports custom branding, your organization might be using a branded build.
```